planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -573,6 +576,11 @@ public Planetiler fetchWikidataNameTranslations(Path defaultWikidataCache) {
         fetchWikidata);
     useWikidata = fetchWikidata || arguments.getBoolean("use_wikidata", "use wikidata translations", true);
     wikidataNamesFile = arguments.file("wikidata_cache", "wikidata cache file", defaultWikidataCache);
+    wikidataMaxAge =
+      arguments.getDuration("wikidata_max_age",
+        "Maximum age of Wikidata translations (in ISO-8601 duration format PnDTnHnMn.nS; 0S = disabled)", "P30D");
+    wikidataUpdateLimit = arguments.getInteger("wikidata_update_limit",
+      "Limit on how many old translations to update during one download (0 = disabled)", 0);
```

Comment:
It would be nice to only require one CLI parameter to enable this, it seems like the main one someone would want to control is the max age. Could we go back to the old update limit but set max age to 0 by default so you just need to set `--wikidata-max-age=P30D` to enable?

Reply:
Adjusted.